From fada68e9f922a3a6a176092a7abdeec0b58408e8 Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Thu, 17 Oct 2013 19:49:19 +0200
Subject: [PATCH] static-nodes: don't call mkdir

This is no longer necessary with kmod-15. Bump the requirement.
---
 README                             | 2 +-
 configure.ac                       | 2 +-
 units/kmod-static-nodes.service.in | 1 -
 3 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/README b/README
index b39cd37..cf0a18d 100644
--- a/README
+++ b/README
@@ -95,7 +95,7 @@ REQUIREMENTS:
         dbus >= 1.4.0
         libcap
         libblkid >= 2.20 (from util-linux) (optional)
-        libkmod >= 14 (optional)
+        libkmod >= 15 (optional)
         PAM >= 1.1.2 (optional)
         libcryptsetup (optional)
         libaudit (optional)
diff --git a/configure.ac b/configure.ac
index 4f26092..5bc31c5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -269,7 +269,7 @@ AC_ARG_ENABLE(kmod, AS_HELP_STRING([--disable-kmod], [disable loadable modules s
 if test "x$enable_kmod" != "xno"; then
         PKG_CHECK_EXISTS([ libkmod ], have_kmod=yes, have_kmod=no)
         if test "x$have_kmod" = "xyes"; then
-                PKG_CHECK_MODULES(KMOD, [ libkmod >= 14 ],
+                PKG_CHECK_MODULES(KMOD, [ libkmod >= 15 ],
                         [AC_DEFINE(HAVE_KMOD, 1, [Define if kmod is available])],
                         AC_MSG_ERROR([*** kmod version >= 14 not found]))
         fi
diff --git a/units/kmod-static-nodes.service.in b/units/kmod-static-nodes.service.in
index ff4017b..368f980 100644
--- a/units/kmod-static-nodes.service.in
+++ b/units/kmod-static-nodes.service.in
@@ -15,5 +15,4 @@ ConditionPathExists=/lib/modules/%v/modules.devname
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStartPre=@MKDIR_P@ /run/tmpfiles.d
 ExecStart=@KMOD@ static-nodes --format=tmpfiles --output=/run/tmpfiles.d/kmod.conf
